docs/DeveloperGuide.md

```diff
@@ -756,19 +739,19 @@ otherwise)
     * 1b1. StoreMando shows an error message.
 
       Use case resumes at step 1.
-
-**Use case: UC6 - Check for expiring items**
+    
+**Use case: UC5 - Check for expiring items**
 
 **MSS**
 
-1. User wants to search for items that are expiring within the next 7 days.
-2. StoreMando displays a list of items that have either expired or are expiring within the next 7 days.
+1. User wants to search for items that are expiring within a specific number of days.
+2. StoreMando displays a list of items that have either expired or are expiring within the specified number of days.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. User inputs a negative number.
+* 1a. User inputs a number smaller than -365 or larger than 365.
```

Comment:
I can't comment above but perhaps you should do this for add and edit too for the quantity limit if you are doing it for reminder.